package/MDAnalysis/analysis/pca.py

```diff
@@ -442,14 +503,14 @@ def cumulative_overlap(self, other, i=0, n_components=None):
 
         See also
         --------
-        cumulative_overlap
+        :func:`cumulative_overlap`
```

Comment:
Builds a link to itself (even though labeled func and not meth), needs to be
```suggestion
        :func:`~MDAnalysis.analysis.pca.cumulative_overlap`
```